integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/command/SICreationCommand.scala

```diff
@@ -683,15 +683,18 @@ private[sql] case class CarbonCreateSecondaryIndexCommand(
     // if data type is arrayType, then store the column as its CHILD data type in SI
     if (DataTypes.isArrayType(parentColumnSchema.getDataType)) {
       columnSchema.setDataType(dataType)
+      val colPropMap = new java.util.HashMap[String, String]()
```

Comment:
Please add it to parentColumnSchema.getColumnProperties map

Reply:
ok done